packages/cli-internal/src/commands/push.ts

```diff
@@ -168,8 +168,11 @@ export default class Push extends Command {
           }
         })
 
+        const hasBuilderDefinedBatchingField = fields.filter((f) => f.fieldKey === 'enable_batching').length === 1
```

Comment:
Reason why this should be exactly `1`? nitpick could do

```suggestion
        const hasBuilderDefinedBatchingField = fields.some((f) => f.fieldKey === 'enable_batching')
```